<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModbusClient</class>
 <widget class="QDialog" name="ModbusClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ModbusClient</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="documentMode">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>通话                                              </string>
    </attribute>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>2</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>42</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_1">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>2</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>2</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>2</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPlay">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>170</y>
       <width>95</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>现在
静音</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>2</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>42</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>42</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>42</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>42</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>82</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user11</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>82</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_13">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>82</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user13</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_14">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>82</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user14</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>82</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user15</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_16">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>122</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user16</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_17">
     <property name="geometry">
      <rect>
       <x>97</x>
       <y>122</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user17</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_18">
     <property name="geometry">
      <rect>
       <x>193</x>
       <y>122</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user18</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_19">
     <property name="geometry">
      <rect>
       <x>289</x>
       <y>122</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user19</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>385</x>
       <y>122</y>
       <width>95</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>user20</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>设置                                             </string>
    </attribute>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>193</y>
       <width>40</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>421</width>
       <height>21</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="acceptRichText">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAdd">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>60</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditValue">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>0</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="volueAdd">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>98</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>音量+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="volueLess">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>98</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>音量-</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>71</x>
       <y>137</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>音量</string>
     </property>
    </widget>
    <widget class="QPushButton" name="testbtnfivesecond">
     <property name="geometry">
      <rect>
       <x>418</x>
       <y>170</y>
       <width>60</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>5s测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="jiangzao">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>98</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_versionnum">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>220</y>
       <width>140</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
